install.sh

```diff
@@ -55,68 +36,28 @@ function install_seed_voicecard_driver()
     cd seeed-voicecard
     sudo ./install.sh
     cd ..
-    mv seeed-voicecard ~/seeed-voicecard
-}
-
-function install_flite_from_source()
-{
-    wget http://www.festvox.org/flite/packed/flite-2.0/flite-2.0.0-release.tar.bz2
-    tar xf flite-2.0.0-release.tar.bz2
-    cd flite-2.0.0-release
-    ./configure
-    make
-    sudo make install
-    cd ..
-    rm -rf flite-2.0.0-release*
+    tar czf ~/seeed-voicecard.tar.gz seeed-voicecard
+    rm -rf seeed-voicecard
 }
 
 function install_dependencies()
 {
     install_seed_voicecard_driver
-    if /usr/bin/dpkg --search /usr/bin/dpkg
-    then
-        sudo -E apt install -y libatlas-base-dev
-    else
-        return 1;
-    fi
 }
 
-function install_snowboy()
-{
-    if install_dependencies
-    then
-        root_dir=$(pwd)
-        sudo pip3 install git+https://github.com/Kitt-AI/snowboy.git
-        if [ $? -ne 0 ]; then
-            echo "FAILED: Unable to make Snowboy Detect file. Please follow manual instructions at https://github.com/kitt-AI/snowboy"
-            echo "You may also use PocketSphinx Detector if you are unable to install snowboy on your machine"
-        else
-            echo "Snowboy Detect successfully installed"
-        fi
-        cd "$root_dir"
-        rm -rf snowboy
-    else
-        echo "FAILED: Installation of Snowboy on your system is not supported presently. Please follow manual instructions at https://github.com/kitt-AI/snowboy"
-    fi
-}
-
-
-
 function susi_server(){
     if  [ ! -d "susi_server" ]
     then
         mkdir $DIR_PATH/susi_server
         cd $DIR_PATH/susi_server
-        git clone https://github.com/fossasia/susi_server.git
-        git clone https://github.com/fossasia/susi_skill_data.git
+        git clone --recurse-submodules https://github.com/fossasia/susi_server.git
+        git clone --recurse-submodules https://github.com/fossasia/susi_skill_data.git
```

Comment:
@hongquan , we do not need to use `--recursive-submodules` for susi_skill_data repo